package-lock.json

```diff
@@ -2130,7 +2130,7 @@
     },
     "jsonfile": {
       "version": "2.4.0",
-      "resolved": "https://registry.npmjs.org/jsonfile/-/jsonfile-2.4.0.tgz",
+      "resolved": "https://nexus3.linkurious.net/repository/npm/jsonfile/-/jsonfile-2.4.0.tgz",
```

Comment:
@yigitakbulut can you revert that? this is a public github repo, it should not reference our private npm repo